organisaatio-ui/src/components/Taulukot/NormaaliTaulukko.tsx

```diff
@@ -0,0 +1,296 @@
+import * as React from 'react';
+import styles from './NormaaliTaulukko.module.css';
+import {
+    useGlobalFilter,
+    useExpanded,
+    usePagination,
+    useTable,
+    useSortBy,
+    useFilters,
+    Column,
+    TableInstance,
+} from 'react-table';
+import Button from '@opetushallitus/virkailija-ui-components/Button';
+import Input from '@opetushallitus/virkailija-ui-components/Input';
+import searchIcon from '@iconify/icons-fa-solid/search';
+import Select from '@opetushallitus/virkailija-ui-components/Select';
+import { Icon } from '@iconify/react';
+import chevronLeft from '@iconify/icons-fa-solid/chevron-left';
+import chevronRight from '@iconify/icons-fa-solid/chevron-right';
+import { useContext } from 'react';
+import { KoodistoContext, LanguageContext } from '../../contexts/contexts';
+import { useState } from 'react';
+import { Ryhma, SelectOptionType, YhteystietoTyyppi } from '../../types/types';
+import { ValueType } from 'react-select';
+
+// TODO yhtenäistä!
+const mapPaginationSelectors = (index) => {
+    if (index < 3) return [0, 5];
+    return [index - 2, index + 3];
+};
+
+export type Props = {
```

Comment:
rather lenient typing?
being more strict _will_ make life easier

Reply:
will change this to NormaaliTaulukkoProps or something similar